11 automated task processing/OrderBooking.WebAPI/Program.cs

```diff
@@ -1,11 +1,18 @@
+using Azure.Search.Documents;
 using MessageHandler.EventSourcing;
 using MessageHandler.EventSourcing.AzureTableStorage;
+using MessageHandler.EventSourcing.DomainModel;
 using MessageHandler.EventSourcing.Outbox;
+using MessageHandler.EventSourcing.Projections;
 using MessageHandler.Runtime;
 using MessageHandler.Runtime.AtomicProcessing;
+using NotificationPreferences;
 using OrderBooking.Projections;
 using OrderBooking.WebAPI;
+using OrderBooking.WebAPI.Controllers;
 using OrderBooking.WebAPI.SignalR;
+using OrderAggregate = OrderBooking.OrderBooking;
+using NotificationAggregate = NotificationPreferences.NotificationPreferences;
```

Comment:
The two lines above can be removed, not used anymore

Reply:
What about MinimalApiConfig, any ideas of renaming it?

Reply:
Perhaps call it RoutingConfiguration?

Reply:
Sounds great